app/views/projects/import.html.erb

```diff
@@ -8,55 +8,70 @@
   <%= render 'projects/navbar', project: @project %>
 <% end %>
 
-<div class="row">
-  <div class="col-sm-12 col-md-12">
-    <h2><%= t('projects.uploads.title') %></h2>
+<%= render partial: 'settings_sidebar',
+           layout: 'sidebar_wrapper' %>
 
-    <% if session[:import_job] %>
-      <%= raw t('projects.uploads.already_uploaded', time_ago: time_ago_in_words( session[:import_job][:created_at] )) %>
-    <% else %>
-      <%= raw t('projects.uploads.instructions') %>
+<div class="settings-page">
+  <div class="col-xs-12 col-sm-9">
+    <h3><%= t('projects.uploads.title') %></h3>
+    <hr>
 
-      <%= form_for [:import_upload, @project], html: { class: "form-horizontal center" } do |f| %>
-        <div class="field">
-          <%= f.attachinary_file_field :import %>
-        </div>
-        <div class="field">
-          <%= f.submit 'import', class: 'btn btn-default btn-primary' %>
-        </div>
-      <% end %>
-    <% end %>
-
-    <% if @valid_stories %>
-      <h2><%= t('projects.uploads.results') %></h2>
+    <div class="card panel">
+      <div class="panel-content">
+        <% if session[:import_job] %>
```

Comment:
I know you didn't changed this part, but you could move it to the controller and create a `@import_job` there, the view should not be aware of the existence of the `session`.